### PR TITLE
feat: added filter page

### PR DESCRIPTION
**Pull request by rdshinde**
Added filter page with scrollable and fixed sidebar.

Screenshot 2022-02-18 at 12 52 21 PM

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -96,6 +95,95 @@ a .fa-store:hover {
...
+  max-height: 35rem;
+  max-width: 18rem;
+  transition: all 0.2s ease-in;
+}
+
+.cards-container .card:hover {
+  transform: scale(1.01);
+}
+
+.cards-container .card .card__footer {
```
Comment (vivekmehto) [inline style.css:160]: we can create utility class  for flex and use it

File: style.css [hunk 2]
```diff
@@ -96,6 +95,95 @@ a .fa-store:hover {
...
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.cards-container .card .card__footer .btn {
```
Comment (vivekmehto) [inline style.css:167]: keep specificity low. or add one utility class for full width;

**Review by vivekmehto — APPROVED** [review]
few suggestions apart from that everything seems good. you can move ahead and change it later.

File: style.css (hunk 2 above)
Comment (rdshinde) [reply style.css:167]: OK.
Thank you

File: style.css [hunk 3]
```diff
@@ -96,6 +95,95 @@ a .fa-store:hover {
   color: var(--primary-color);
 }
 
+
+/* Aside Container */
```
Comment (ImrozNezami) [inline style.css:100]: remove comment

File: style.css [hunk 4]
```diff
@@ -96,6 +95,95 @@ a .fa-store:hover {
...
+  grid-area: main;
+  margin: auto;
+  width: 100%;
+}
+
+.main-container {
+  width: 95%;
+}
+
+/* Stling for card component and card-container */
```
Comment (ImrozNezami) [inline style.css:142]: remove comment

File: style.css [hunk 5]
```diff
@@ -96,6 +95,95 @@ a .fa-store:hover {
...
+}
+
+.card__wishlist {
+  font-size: calc(var(--scale-md) + var(--scale-sm));
+}
+
+.card__wishlist:active {
+  color: var(--warning-color);
+}
 /* Hero Styling */
```
Comment (ImrozNezami) [inline style.css:188]: remove comment

**Pull request merged**